src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/Invertible.java

```diff
@@ -0,0 +1,8 @@
+package io.stargate.sgv2.jsonapi.api.model.command.clause.filter;
+
+/** Interface Invertible implemented by comparisonExpression and logicalExpression */
+public interface Invertible {
+
+  /** invert method to invert a comparisonExpression or logicalExpression */
+  Object invert();
```

Comment:
Return should be `Invertible` instead of `Object`

Reply:
Thanks! resolved.